questions/qa-direction-native.en.html

```diff
@@ -14,13 +14,13 @@
 f.previousauthors = ''; // as above
 f.modifiers = ''; // people making substantive changes, and their affiliation
 f.searchString = 'qa-direction-native'; // blog search string - usually the filename without extensions
-f.firstPubDate = '2024-07-16'; // date of the first publication of the document (after review)
-f.lastSubstUpdate = { date:'2024-07-16', time:'07:07'}  // date and time of latest substantive changes to this document
+f.firstPubDate = '2024-07-04'; // date of the first publication of the document (after review)
```

Comment:
We shouldn't probably change `firstPubDate`?

Reply:
```suggestion
f.firstPubDate = '2024-07-16'; // date of the first publication of the document (after review)
```